app/controllers/support_interface/provider_users_controller.rb

```diff
@@ -13,7 +13,8 @@ def index
     end
 
     def show
-      redirect_to edit_support_interface_provider_user_path(params[:id])
+      provider_user = ProviderUser.find(params[:id])
+      @form = ProviderUserForm.from_provider_user(provider_user)
```

Comment:
I think we should pass `@provider_user` instead of form — confusing to have a `@form` on a show page